### PR TITLE
Fix analytics (groups & users)

### PR DESCRIPTION
**Pull request by Donorhan**
- Fix issue with groups
- Send analytic event when the user update his profile
- Add group to the following events: Sign In & Log Out

### CODE REVIEW

**Review by xsyann — APPROVED** [review]

File: core/server/analytics.js [hunk 1]
```diff
@@ -155,7 +137,12 @@ Meteor.methods({
     });
 
     const user = Meteor.user();
-    analytics.track(userId, '🎤 Conference Attend', { level_id: user.profile.levelId, zone_id: traits.zoneId, zone_name: traits.zoneName });
+    analytics.track(userId, '🎤 Conference Attend', {
+      guild_id: user.guildId,
```
Comment (ramnes) [inline core/server/analytics.js:141]: Why tracking the `guild_id` here and in other similar events? It seems to me that it is the kind of information that belongs to the user and thus to `identify` calls. :)

Comment (Donorhan) [reply core/server/analytics.js:141]: From what I've seen in the documentation, adding the attribute of a group to an event allows to link an event to a group (in addition to linking it to an user)
Capture d’écran, le 2022-09-01 à 17 30 01

Comment (ramnes) [reply core/server/analytics.js:141]: Okay, weird but right, seems to be the Mixpanel way indeed. TIL!

Comment (Donorhan) [reply core/server/analytics.js:141]: I have the feeling that a tool like June is smart enough to automatically make the link between event -> user -> group, while tools like mixpannel don't make the link between the data, at least that's what I noticed with my tests

**Pull request merged**